test/run

```diff
@@ -794,14 +789,6 @@ testFailingBuild() {
   assertCapturedError 1 ""
 }
 
-testTicketOnFailure() {
-  compile "invalid-dependency"
-  assertCaptured "troubleshooting-node-deploys"
-  assertCaptured "please submit a ticket"
-  assertNotCaptured "possible problems"
-  assertCapturedError
-}
```

Comment:
moved these assertions so that invalid-dependency does not need to be compiled twice